src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanOrchestrator.hpp

```diff
@@ -71,7 +71,13 @@ class TScanOrchestrator final : public TOSPrimitives
                                std::shared_mutex& mutex)
         : m_mutex {mutex}
     {
-        m_inventoryDatabase = std::make_unique<Utils::RocksDBWrapper>(INVENTORY_DB_PATH);
+        bool repaired {false};
+        m_inventoryDatabase = std::make_unique<Utils::RocksDBWrapper>(
+            Utils::RocksDBWrapper::openAndRepairBuilder(INVENTORY_DB_PATH, repaired));
```

Comment:
An alternative could be to return a `std::pair<std::unique_ptr<Utils::RocksDBWrapper>, bool>` instead of passing the repaired as an argument

https://www.fluentcpp.com/2021/07/09/how-to-return-several-values-from-a-function-in-c/

Reply:
Now the method returns a tuple-